commnad_line_rock_paper_scissors.py

```diff
@@ -6,6 +6,27 @@
 from rock_paper_scissors.game import Game
 
 
+def human_input_to_move(game, human):
+    choices = ', or '.join(
+        [
+            ', '.join(game.valid_choices[:-1]),
+            game.valid_choices[-1]
+        ]
+    )
+
+    try:
+        human.choice = game.convert_input_to_move(input('Choose {}: '.format(choices)))
+    except Exception:
+        print('Input is invalid')
```

Comment:
If you implement my comment above, you'll notice that `human` isn't actually getting used in this function (other than being passed to the same function when an exception is caught). The reason I suggested including a `user` in the inputs to this function was so that the printed error message could reference the name of the user who provided the wrong input (like you were previously doing [here](https://github.com/briana-mcn/rock-paper-scissors/pull/15/commits/7d422cda7c463ffe7074dd96fedf04276535f5f7#diff-23e75bcf3f5dc13f1b949fa5243be3e0R42)).

Something like:
```python
print('Input from {} is invalid, enter another choice'.format(human))
```

However, if you prefer your error message the way it currently is (`'Input is invalid'`), then you can simply remove `human` from this function altogether.